pkg/cluster/cluster_test.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	. "github.com/onsi/ginkgo"
```

Comment:
Maybe a nit but since these functions as related to cluster IDs perhaps we could name the files accordingly, ie _id.go_ and _id_test.go_.